frontend/src/main/scala/bloop/bsp/BloopBspServices.scala

```diff
@@ -863,6 +864,20 @@ final class BloopBspServices(
   private def toBuildTargetId(project: Project): bsp.BuildTargetIdentifier =
     bsp.BuildTargetIdentifier(project.bspUri)
 
+  private def toJvmBuildTarget(project: Project): bsp.JvmBuildTarget = {
+    Option(project.platform)
+      .collect {
+        case jvm: Platform.Jvm =>
+          val javaHome = bsp.Uri(jvm.config.javaHome.toBspUri)
+          bsp.JvmBuildTarget(Some(javaHome), None)
+      }
+      .getOrElse {
+        val javaHome = Option(System.getProperty("java.home")).map(s => bsp.Uri(Paths.get(s).toUri))
+        val javaVersion = Option(System.getProperty("java.vm.specification.version"))
+        bsp.JvmBuildTarget(javaHome, javaVersion)
+      }
```

Comment:
Isn't this method returning a jvm build target always, regardless of the platform?

Reply:
Why return a `JvmBuildTarget` here for projects that are not JVM?